subsys/app_event_manager/app_event_manager_priv.h

```diff
@@ -63,6 +63,13 @@ extern "C" {
 #define _APP_EM_SUBS_PRIO_EARLY  05
 #define _APP_EM_SUBS_PRIO_NORMAL 10
 
+/* Size setting enum state generator, required for inter-core compatibility.
+ * Shall be called as last state of enum.
```

Comment:
s/Shall be called as last state of enum./Shall be added as the last element of enum.